src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -3082,23 +3082,17 @@ void TLayout::layout(HarmonicMarkSegment* item, LayoutContext& ctx)
     Autoplace::autoplaceSpannerSegment(item, ldata, ctx.conf().spatium());
 }
 
-void TLayout::layout(Harmony* item, LayoutContext& ctx)
+void TLayout::layout(const Harmony* item, Harmony::LayoutData* ldata, LayoutContext& ctx)
 {
     if (!item->explicitParent()) {
```

Comment:
Are the several `item->explicitParent()` checks in this method still needed?

Reply:
unfortunately, yes (
layout method called from constructor... 
I tried to fix it somehow simple, but it didn't work, I need to figure it out and redo it.

![image](https://github.com/musescore/MuseScore/assets/3818029/6761e0bf-1acb-40c5-9ec6-c29abb1aebca)